### PR TITLE
(Feature) Manage created crowdsale contracts - display real data

### PR DESCRIPTION
**Pull request by fernandomg**
This PR is the third step towards in the completion of #323

- Crowdsale address has to be manually added to the `CrowdsaleStore` to be able to load `manage/` url
- Information being displayed is the information that can be retrieved from the blockchain

### CODE REVIEW

**Comment by fvictorio** [comment]
I'm getting a `TypeError: Cannot set property 'rate' of undefined` error when I input the rate in step three.

**Comment by vbaranov** [comment]
@fernandomg, please, resolve conflicts. They appear after merging https://github.com/poanetwork/ico-wizard/pull/407.

File: src/components/Common/CrowdsalesList.js [hunk 1]
```diff
@@ -29,20 +29,20 @@ export default class CrowdsalesList extends Component {
         <div className="container-fluid">
           <div className="table-row flex-table-header">
             <div className="text">Address</div>
-            <div className="sm-text">Token Name</div>
-            <div className="sm-text">Ticker</div>
-            <div className="sm-text">Date</div>
+            {/*<div className="sm-text">Token Name</div>*/}
```
Comment (vbaranov) [inline src/components/Common/CrowdsalesList.js]: @fernandomg can we delete these commented lines before merging to master?

File: src/components/Common/CrowdsalesList.js [hunk 2]
```diff
@@ -29,20 +29,20 @@ export default class CrowdsalesList extends Component {
...
                    key={index.toString()}
-                   onClick={() => this.selectCrowdsale(index, crowdsale.contractAddress)}
+                   onClick={() => this.selectCrowdsale(index, crowdsaleAddress)}
               >
-                <div className="text">{crowdsale.contractAddress}</div>
-                <div className="sm-text">{crowdsale.extraData.tokenName}</div>
-                <div className="sm-text">{crowdsale.extraData.ticker}</div>
-                <div className="sm-text">{crowdsale.extraData.creationDate.split('T').join(' ')}</div>
+                <div className="text">{crowdsaleAddress}</div>
+                {/*<div className="sm-text">{crowdsaleAddress.extraData.tokenName}</div>*/}
```
Comment (vbaranov) [inline src/components/Common/CrowdsalesList.js]: @fernandomg can we delete these commented lines before merging to master too?

File: src/stores/CrowdsaleStore.js [hunk 3]
```diff
@@ -2,421 +2,35 @@ import { observable } from 'mobx'
...
+      '0x005364854d51a0a12cb3cb9a402ef8b30702a570',
+      '0x005364854d51a0a12cb3cb9a402ef8b30702a571',
+      '0x005364854d51a0a12cb3cb9a402ef8b30702a572'
     ])
+    this.selected = {
+      updatable: true
+    }
+  }
+
+  @action setProperty = (property, value) => {
```
Comment (vbaranov) [inline src/stores/CrowdsaleStore.js]: @fernandomg there is no action decorator imported from `mobx` package here.

File: src/components/Common/CrowdsalesList.js (hunk 1 above)
Comment (fernandomg) [reply src/components/Common/CrowdsalesList.js]: removed

File: src/components/Common/CrowdsalesList.js (hunk 2 above)
Comment (fernandomg) [reply src/components/Common/CrowdsalesList.js]: removed

File: src/stores/CrowdsaleStore.js (hunk 3 above)
Comment (fernandomg) [reply src/stores/CrowdsaleStore.js]: added `action`

**Pull request merged**